hw/ip/pwrmgr/dv/cov/pwrmgr_cov_unr_excl.el

```diff
@@ -0,0 +1,195 @@
+// Copyright lowRISC contributors.
```

Comment:
This needs design review, and is significantly more complex. It would be great if some of these could be excluded in code.